### PR TITLE
Reorder categories and round WU values

### LINKED ISSUE #199: Reorder categories
re-order categories to thermo, irrigation, public supply, industrial

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Fixes #199 and #193

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
image

**Comment by ldecicco-USGS** [comment]
Those numbers look like "321672.07"...I guess those are the numbers going on the legend when we're not hovered on a county? What do you think we should shoot for here, integers?

**Comment by aappling-usgs** [comment]
Response on Slack is to stick to hundredths for now, then consult with the NWULT team about what would be acceptable to them.

**Comment by ldecicco-USGS** [comment]
Warning...I'm about to push a few things up...so I can better see what I need to remove from this pr to make it a finished product. So...expect 2 pr's...then it should be ready to merge....

**Comment by ldecicco-USGS** [comment]
I think it's finally working:

firstgif

**Comment by aappling-usgs** [comment]
Wow, and you got commas in there and everything! :tada:

File: scripts/process/state_wu_data.R [hunk 1]
```diff
@@ -4,16 +4,31 @@ process.state_wu_data <- function(viz) {
...
-    dplyr::rename(abrv=STATE) %>% 
-    dplyr::arrange(wu)
-  
-  # calculate other category value
+  state_data <- list()
+  i <- 1
+  for(state in unique(wu_df$STATE)){
+    
+    wu_df_state <- wu_df %>%
+      dplyr::filter(STATE == state) %>%
```
[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply scripts/process/state_wu_data.R]: @aappling-usgs if you've got any suggestions on how to get this json file to save without the excessive []'s ....this is the place it's happening.

[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply scripts/process/state_wu_data.R]: The data currently looks like:
```
[{"abrv":["DC"],
"open":[false],
"STATE_NAME":["District of Columbia"],
"use":[{"category":"total","wateruse":0.05},
{"category":"thermoelectric","wateruse":0},
{"category":"publicsupply","wateruse":0},
{"category":"irrigation","wateruse":0.05},
{"category":"industrial","wateruse":0}]},
{"abrv":["VT"],
"open":[false],
"STATE_NAME":["Vermont"],
"use":[{"category":"total","wateruse":90.94},
{"category":"thermoelectric","wateruse":0.8},
{"category":"publicsupply","wateruse":42.66},
{"category":"irrigation","wateruse":3.11},...
```
we'd like it to look (I think) like:
```
[{"abrv":"DC",
"open":false,
"STATE_NAME":"District of Columbia",
"use":[{"category":"total","wateruse":0.05},
{"category":"thermoelectric","wateruse":0},
{"category":"publicsupply","wateruse":0},
{"category":"irrigation","wateruse":0.05},
{"category":"industrial","wateruse":0}]},
{"abrv":"VT",
"open":false,
"STATE_NAME":"Vermont",
"use":[{"category":"total","wateruse":90.94},
{"category":"thermoelectric","wateruse":0.8},
{"category":"publicsupply","wateruse":42.66},
{"category":"irrigation","wateruse":3.11},...
```

File: js/map_utils.js [hunk 2]
```diff
@@ -271,35 +271,41 @@ function updateLegendText(d, category) {
...
+      .data(waterUseViz.nationalData, function(d) { return d.category; })
+      .text(function(d) { return d.fancynums; });
+
   } else {
-    console.log('activeView is not USA, so not yet setting category amounts because this is just a test');
+
+    
+   var state_data = waterUseViz.stateData
+      .filter(function(d) { 
+        return d.abrv == activeView;
```
[the remark this answers is not in the document]
Comment (aappling-usgs) [reply js/map_utils.js]: ohh, maybe so. i'll play with this for a sec.

File: js/national_pie.js [hunk 3]
```diff
@@ -1,43 +1,41 @@
...
+      .innerRadius(radius*1.15);
   
-  var wu_national_no_total = wu_national_data
+  // for pie slice text placement
+  var textArc = d3.arc()
+      .outerRadius(radius*1.25)
+      .innerRadius(radius*1.25);
+  
+  var wu_national_no_total = waterUseViz.nationalData  
         .filter(function(d) { return d.category !== "total"; });
```
[the remark this answers is not in the document]
Comment (aappling-usgs) [reply js/national_pie.js]: :+1:

File: scripts/process/state_wu_data.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (aappling-usgs) [reply scripts/process/state_wu_data.R]: I will look right now

**Comment by aappling-usgs** [comment]
See https://github.com/ldecicco-USGS/water-use-15/pull/1

**Comment by aappling-usgs** [comment]
Also finishes https://github.com/USGS-VIZLAB/water-use-15/issues/181

**Pull request merged**